src/utils/num.ts

```diff
@@ -176,6 +301,11 @@ export function hexToBytes(value: string): Uint8Array {
  * @param number value to be increased
```

Comment:
Precise the type

Reply:
changed "@param number value to be increased"  
to:
" @param number The value to be increased (BigInt or number)."